.chloggen/prwreceiver-checkscope.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: receiver/prometheusremotewrite
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Check if Scope is already present comparing with the received labels
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: [35656]
```

Comment:
```suggestion
issues: [36927]
```

Reply:
But this number is not related to any issue. Using it, we will point to this PR, is that right?

Reply:
Yep! See comment on line 12